Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -89,7 +89,7 @@ public class SchemeGenerator {
             switch buildTarget.target.location {
             case .project(let project):
                 guard let projectReference = self.project.getProjectReference(project) else {
-                    throw SchemeGenerationError.missingProject(project)
+                    fatalError("Unable to find project reference named \"\(project)\" in project.yml")
```

Comment:
Sorry for confusing you. I think it's better L92 and L102 should be still throwing exceptions.
(I agreed to revert L114 only)

The reason why, these(L92 and L102) are user errors, which they would be caused by user inputs.
If users mistook the project names, these errors are raised in regular situations.

There is one more reason. `func generateScheme` is a public throwable function. function users expect to throw errors by wrong usages.
So I think you're better to throw exceptions instead of calling `fatalError`.

How do you think about my opinion? @kateinoigakukun @yonaskolb